controller/src/test/java/io/pravega/controller/server/v1/ControllerServiceTest.java

```diff
@@ -144,11 +144,11 @@ public void setup() throws Exception {
         startTs = System.currentTimeMillis();
         OperationContext context = streamStore.createContext(SCOPE, stream1);
         streamStore.createStream(SCOPE, stream1, configuration1, startTs, context, executor).get();
-        streamStore.setState(SCOPE, stream1, State.ACTIVE, context, executor);
+        streamStore.setState(SCOPE, stream1, State.ACTIVE, context, executor).get();
```

Comment:
This is unrelated to the rest of the change but fixes a flakey test.

Reply:
ok